frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -32,22 +33,23 @@ const ServicesContext = createContext<ServicesContextProps>(null);
 const handleError = (
   error: AxiosError,
   t: (key: string) => string,
-  meta: QueryMeta | MutationMeta
+  meta: QueryMeta | MutationMeta,
+  logout: () => Promise<void>
 ): void => {
   // TODO : log axios errors
   // TODO : handle messages from API
-  // TODO : logout user when session is expired
-
   if (
     meta?.hideDefaultError === true ||
     (meta?.hideDefaultError instanceof Function && meta?.hideDefaultError?.(error))
   )
     return;
 
-  toast.error(
-    () => t('general.error_message'),
-    { toastId: 'default' }
-  );
+  if (error?.response?.status === 401 || error?.response?.status === 403) {
```

Comment:
Are we entirely certain that we should check for a 403 here? 401 means "Unauthorized," whereas 403 signifies "Forbidden" but you are logged in.

Reply:
Good point, I removed it 👍